script/main.js

```diff
@@ -76,10 +76,10 @@ async function updateNotebooks(){
             'Content-type': 'application/json'
         },
         body: JSON.stringify({id: `${idNumber}`, title: titleField, description: textAreaField})
-    });
-
-
-}
+    })
+    const result = await response.json();
+    console.log('Updated notebook: ', result)
```

Comment:
Always remove console logs. 
when making a PR you are saying that this is ready for production. we cant have console.logs there